genisys/server/__init__.py

```diff
@@ -79,3 +89,43 @@ def drop_priviledges(config: ServerOptions) -> pwd.struct_passwd:
     os.setuid(uid.pw_uid)
     os.setgid(gid.gr_gid)
     return uid
+
+# end drop_privledges
+
+def meteor_initialization(server_config: ServerOptions):
+    '''Runs Meteor as a subprocess of Genisys and 
+    initializes necessary environment variables for
+    Meteor. This process assumes that the user already
+    has MongoDB installed and running.'''
+
+    # Set environment variables
+    os.environ['ROOT_URL'] = 'http://localhost'
+    os.environ['PORT'] = '8080'
+    if 'MONGO_URL' not in os.environ:
+        print('MONGO_URL not found in environment variables, cancelling Meteor server.')
+        return
+    print('MONGO_URL found.')
+
+    if 'CONFIG_FILE' not in os.environ:
```

Comment:
Where will CONFIG_FILE get set at? Is it expected that the user will set it before they run `genisys server`? That could be redundant if the user has to do `CONFIG_FILE=/path/to/genisys.yaml genisys server -f /path/to/genisys.yaml`